lib/messenger.rb

```diff
@@ -0,0 +1,22 @@
+module Messenger
+
+  def send_sms(number, content)
+    sid = ENV["TWILIO_ACCT_SID"]
+    auth = ENV["TWILIO_AUTH"]
+
+    @client = Twilio::REST::Client.new sid, auth
+
+    from = "+14123854063 "
+
+    message = @client.account.messages.create(
+             :from => from,
+             :to => '+1'+number,
+             :body => content
+    )
+  end
+
+  def receive_sms
```

Comment:
this isn't implemented, make sure to implement or remove before merging to master